### PR TITLE
update-bunsen-pepperflash: remove Debianisms (if that's a word).

### PR DESCRIPTION
**Pull request by Head-on-a-Stick**
I've made these changes to a local version (for Arch), are they worth merging?

They remove the reliance on `dpkg`, I don't think it breaks anything.

### CODE REVIEW

File: update-bunsen-pepperflash [hunk 1]
```diff
@@ -126,7 +126,7 @@ check_versions(){
         return 0
     fi
 
-    if dpkg --compare-versions "$newver" gt "$instver"
+    if [[ "$newver" > "$instver" ]]
```
Comment (ghost) [inline update-bunsen-pepperflash:129]: Works only if it is guaranteed that the versions are of the form x.y.z. Probably works though.

**Comment by johnraff** [comment]
In two minds here:

Considering this is a Debian package, I think it's safe to take the availability of dpkg for granted, and the dpkg command for version comparison is more robust. 

OTOH, there still might be something to be said for making update-bunsen-pepperflash more portable for repackagers, and the simple sort order comparison works with Adobe's present version number syntax. (NB: 010.0.0 will be sorted lower than 9.0.0)

(Re: function definition syntax: while the spaces between the function name and brackets, and between brackets and opening brace, are both optional, the most common practice seems to be *name() {...* 
and I'd prefer to keep the current formatting for line 109.)

**Comment by Head-on-a-Stick** [comment]
> I'd prefer to keep the current formatting for line 109

All of the other functions in the file do not have a space between the round and curly brackets, it was the lack of consistency that bothered me rather than adherence to any particular standard (OT: I prefer the POSIX _function foo {}_).

Anyway there's another `dpkg` hiding in line 83, I will have to look at that as well.

I have no strong opinion about this suggested merge, I just thought it might be of interest.

**Comment by johnraff** [comment]
> All of the other functions in the file do not have a space between the round and curly brackets,

Good grief, you're (almost) right.  (except for `error_exit() {`) 
I usually put in the space, so I don't know what happened here. Anyway, yes, consistency.

> Anyway there's another dpkg hiding in line 83, I will have to look at that as well.

That's my finally chosen way to test if a package is installed on a Debian system (after trying various others). You'll have to find an Arch version of course, or just drop line 83. Really, the dependencies should make it unnecessary in Debian: `Conflicts: pepperflashplugin-nonfree
Replaces: pepperflashplugin-nonfree` so you'll need to think about protecting it from clashes with pepperflashplugin-nonfree in Arch - if such a package exists.

Considering #4 and the possible rewriting of the script, let's leave the current version as it is for now, but bear portability in mind in future.

**Comment by Head-on-a-Stick** [comment]
> let's leave the current version as it is for now

OK, I will go ahead and close this one, thanks for the input folks.

**Pull request closed without merging**